plugins/destination/clickhouse/queries/tables_test.go

```diff
@@ -17,7 +17,7 @@ func TestCreateTable(t *testing.T) {
 			schema.Column{
 				Name:            "extra_col",
 				Type:            schema.TypeFloat,
-				CreationOptions: schema.ColumnCreationOptions{PrimaryKey: true},
+				CreationOptions: schema.ColumnCreationOptions{NotNull: true},
```

Comment:
No need to test primary keys for a destination that supports only append mode.
We do however need a not null key as we using as a sorting key